version/description

```diff
@@ -1,11 +1,7 @@
-v0.4.0
-Bump dependencies to align with kubernetes v1.19.1
-
+v0.5.0
 Features:
-* Bump device-plugin-manager and kubelet deps (#30)
-* Bump dependencies (#29)
-Bugs:
-
+* tier1, Change tests to get namespace from cluster
+* tier1, Change KUBECONFIG to be configurable in functest
 ```
-docker pull quay.io/kubevirt/macvtap-cni:v0.4.0
+docker pull HANDLER_IMAGE
```

Comment:
AFAIK there is a bug/missing feature in release scripts - we never replace this placeholder. Please put in the image name manually.

Reply:
right. 
[Done](https://github.com/kubevirt/macvtap-cni/compare/684314a4bcc23a7a47b5b56e80d5f714fa5bbe90..cf08f54ff1b38b277c9f3a609f776f1c5a21045f)